### ISSUE TITLE
22805882: Voiceover in SpriteKit scene fails changing focus on nodes

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#6259]
#### Description

Summary:
User can't change voiceover focus inside a SKScene. The voiceover focus is kept always in the last node added in the scene. Also user can select the status bar elements when scene is fullscreen. isAccessibleElement property and node focus not working as expected.

Steps to Reproduce:
Create a game project in Xcode using template and SpriteKit.
Have at least 2 nodes in the scene.
Run game with voiceover enable (enabling/disabling during game makes no difference)
Try to change focus on nodes, focus can't be changed. It always stays in the last node added in the scene with isAccessibleElement property set as true.

Expected Results:
User should be able to change focus on nodes in scene so voiceover can read the the 'hint' and 'label' properties set when isAccessibleElement property is set 'true'.

Actual Results:
User can't change focus among node elements in scene when isAccessibleElement property is set 'true'. Focus persists in only one element (last added in scene with isAccessibleElement property set as true).

Version:
9.0 (13A344)

Notes:
User can also access the status bar while in a fullscreen SKScene by swipe up right top corner of the screen or click in an area where the status bar should be. Then user can't go back to scene to select any nodes, ends up stuck in status bar.

Configuration:
SpriteKit game using latest Xcode and Swift 2.0 targetting iOS 9.
## 

Product Version: iOS 9.0 (13A344)
Created: 2015-09-30T13:53:02.317830
Originated: 2015-09-22T16:43:00
Open Radar Link: http://www.openradar.me/22805882
